src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  padding: 0;
 }
+
+.logo img {
+  max-height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  font-weight: 500;
+  min-width: 1024px;
+  position: relative;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0, 5);
```

Comment:
The box-shadow property has an incorrect value 'rgba(0, 0, 0, 0, 5)'. The last parameter should be a decimal between 0 and 1 to represent opacity. Consider correcting this to achieve the desired shadow effect.